src/sagas/favorites_sagas.js

```diff
@@ -0,0 +1,124 @@
+import { call, takeEvery, put, takeLatest, select } from 'redux-saga/effects';
+import actions from '../actions/index';
+import {
+  GET_FAVORITES,
+  LOAD_FAVORITESR,
+  FILTER_CHANGE,
+  ADD_REMOVE_FAVORITE,
+  ADD_REMOVE_FAVORITE_HOME,
+  ADD_REMOVE_FAVORITE_VIEW,
+  ADD_REMOVE_FAVORITE_DETAILS_R
+} from '../actions/types';
+import { getPaginator } from '../utils/general';
+import selectors from '../utils/selectors';
+
+const lim = 15;
```

Comment:
I understand this variable, but to improve readeablility I would call it `const LIMIT = 15;`, because it is an actual constant so it should be uppercase, and because it describes better what is this constant.